bin/node/src/commands/genesis/inputs.rs

```diff
@@ -39,7 +41,37 @@ pub struct BasicFungibleFaucetInputs {
     pub storage_mode: String,
 }
 
-#[derive(Debug, Clone, Copy, Deserialize)]
+#[derive(Debug, Clone, Copy, Deserialize, Serialize)]
 pub enum AuthSchemeInput {
     RpoFalcon512,
 }
+
+impl Default for GenesisInput {
+    fn default() -> Self {
+        const SEED_SUFFIX: &str = "123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef";
```

Comment:
Let's add a brief comment about this suffix (i.e., why we have it).

Reply:
After some discussion we decided to remove it as it didn't have a specific purpose and it may cause some confusion (like thinking the seed must end with that suffix).